lib/src/components/NxCheckbox/NxCheckbox.tsx

```diff
@@ -56,7 +56,7 @@ const NxCheckbox = forwardRef<HTMLLabelElement, Props>(
             {/* Put a space in the box if not checked,
               * in order to provide a consistent vertical-align baseline
               */}
-            { isChecked ? <FontAwesomeIcon icon={faCheck} /> : '\u00A0' }
+            <FontAwesomeIcon icon={faCheck} />
```

Comment:
Two things: is this in scope? if so do we still need the comment above?

Reply:
> Two things: is this in scope? if so do we still need the comment above?

Hi Glen, I believe so. This change is needed because the icon needs to be visible in the `active` state as well as the`checked` state according to Kaleido. Nice catch, I forgot to remove the comment.

Reply:
> > Two things: is this in scope? if so do we still need the comment above?
> 
> Hi Glen, I believe so. This change is needed because the icon needs to be visible in the `active` state as well as the`checked` state according to Kaleido. Nice catch, I forgot to remove the comment.

Well technically the `:active` state of a checkbox (does a checkbox even have an `active` state?) could be either of the state of checking it or unchecking it. The checkmark icon should only appear when the checkbox is in the `:checked` state

Reply:
> > > Two things: is this in scope? if so do we still need the comment above?
> > 
> > 
> > Hi Glen, I believe so. This change is needed because the icon needs to be visible in the `active` state as well as the`checked` state according to Kaleido. Nice catch, I forgot to remove the comment.
> 
> Well technically the `:active` state of a checkbox (does a checkbox even have an `active` state?) could be either of the state of checking it or unchecking it. The checkmark icon should only appear when the checkbox is in the `:checked` state

Yeah, I agree that it should only appear in the `:checked` state. I was confused because in the Kaleido the `:active` state is the same as the `:checked` state. I started a conversation about it in the #react-components channel and it looks like @alexalyons126 will be updating it

Reply:
> > > > Two things: is this in scope? if so do we still need the comment above?
> > > 
> > > 
> > > Hi Glen, I believe so. This change is needed because the icon needs to be visible in the `active` state as well as the`checked` state according to Kaleido. Nice catch, I forgot to remove the comment.
> > 
> > 
> > Well technically the `:active` state of a checkbox (does a checkbox even have an `active` state?) could be either of the state of checking it or unchecking it. The checkmark icon should only appear when the checkbox is in the `:checked` state
> 
> Yeah, I agree that it should only appear in the `:checked` state. I was confused because in the Kaleido the `:active` state is the same as the `:checked` state. I started a conversation about it in the #react-components channel and it looks like @alexalyons126 will be updating it

After offline discussions, the decision was to revert back to how it was before. 44c1e9043dad5474c028b5d79b166e6b7e7baaef